helm/trivy/templates/vpa.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Capabilities.APIVersions.Has "autoscaling.k8s.io/v1" -}}
+{{- if .Values.verticalPodAutoscaler.enabled -}}
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: {{ include "trivy.fullname" . }}-vpa
+  labels:
+    {{- include "trivy.labels" . | nindent 4 }}
+spec:
+  resourcePolicy:
+    containerPolicies:
+    - controlledResources:
+        - cpu
+        - memory
+      maxAllowed:
+        cpu: 1
+        memory: 1Gi
+      minAllowed:
+        cpu: 200m
+        memory: 512Mi
```

Comment:
What are these numbers based on? They seem reasonable but if we're setting actual values, these should be made configurable in `values.yaml` as well.